_pytest/junitxml.py

```diff
@@ -71,6 +71,8 @@ def __init__(self, nodeid, xml):
         self.nodes = []
         self.testcase = None
         self.attrs = {}
+        self.testsuite_properties = {}
```

Comment:
Yes. The _NodeReporter is designed for testcase node.. here i tried to make it represent both. 

I will move it to LogXML and let me know if it will make more sense to you.. 

thank you for the comments.